PluginsAndFeatures/AddLibrary/AzureLibraries/pom.xml

```diff
@@ -0,0 +1,145 @@
+<!--
+  ~ /**
+  ~  * Copyright (c) Microsoft Corporation
+  ~  * <p/>
+  ~  * All rights reserved.
+  ~  * <p/>
+  ~  * MIT License
+  ~  * <p/>
+  ~  * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+  ~  * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+  ~  * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+  ~  * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+  ~  * <p/>
+  ~  * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+  ~  * the Software.
+  ~  * <p/>
+  ~  * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+  ~  * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~  * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+  ~  * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+  ~  * SOFTWARE.
+  ~  */
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.azuretools</groupId>
+    <artifactId>com.microsoft.azuretools.sdk.lib</artifactId>
+    <version>3.0.3</version>
+    <packaging>bundle</packaging>
+    <name>Package for Microsoft Azure Libraries for Java Plugin</name>
+    <organization><name>Microsoft Corp.</name></organization>
+    <parent>
```

Comment:
parent statement here seems inappropriate if module om.microsoft.azuretools.sdk.lib is not part of Utils.